README.md

```diff
@@ -21,32 +39,33 @@ $ psql -f db/schema.sql -d soapbox_dev
 
 2. **Build and install the Soapbox API server (soapboxd) and CLI client (soapboxcli):**
 ``` shell
+$ mkdir -p $GOPATH/src/github.com/adhocteam
+$ ln -s /path/to/soapbox $GOPATH/src/github.com/adhocteam/soapbox
+$ cd $GOPATH/src/github.com/adhocteam/soapbox
+$ go get -u github.com/golang/dep/cmd/dep
+$ dep ensure
```

Comment:
You should not need to do `dep ensure` when installing for the first time, because all the 3rd party dependencies are vendored. You only need to `dep ensure` when you've added removed or modified a 3rd party dependency.